File: app/public_html/script/functions/samples/37.c
loop(){
    if (getHit()){
        speak("Aieeee");
        ambush();
        turnToLastHit();
    }
    else if (getCloseEnemy()){
        attackRanged(getTargetX(), getTargetY());
        speak("Te achei em X:%.1f Y:%.1f", getTargetX(), getTargetY());
    }
    else{
        if(getX() == 12.5 && getY() == 12.5){
            turn(30);
            speak("Roda roda roda...");
        }
        else
            moveTo(12.5,12.5);
    }
}

File: app/public_html/script/functions/samples/37.py
def loop():
    if getHit():
        speak("Aieeee")
        ambush()
        turnToLastHit()
     
    elif getCloseEnemy():
        attackRanged(getTargetX(), getTargetY())
        speak("Te achei em X:{} Y:{}".format(getTargetX(), getTargetY()))
     
    else:
        if getX() == 12.5 and getY() == 12.5:
            turn(30)
            speak("Roda roda roda...")
         
        else:
            moveTo(12.5,12.5)
     
 